### DISCUSSION TITLE
Default versions of lua and luarocks (defaulting to 53 instead of the latest 54)

### DISCUSSION THREAD

**Discussion by aadi58002** [void-linux/void-packages#49554] [General]
I wanted to try out neorg.nvim for neovim and noticed that default version of lua and the luarocks package both point to lua53 instead of the latest lua54.
Just curious as to is there any reason for this choice ?

**Comment by classabbyamp** [comment]
probably because lua53 was installed first, so the alternatives group was set to that version

also, with neovim you probably want to be using luajit

**Comment by aadi58002** [comment]
Thanks for the answer. I believe most users expect to get the latest stable version of lua when they install the lua package. So would such a change be accepted. 

I am aware that neovim uses luajit not the base lua package. What I wanted to try was a extension which depends on the lua rock package. 

I also noticed that luarocks-lua54 package depends on lua53 and luarocks-lua53 package.
